tfhe/src/integer/server_key/radix_parallel/comparison.rs

```diff
@@ -544,7 +544,7 @@ impl ServerKey {
                 } else {
                     // u64::MAX is -1 in tow's complement
                     // We apply the modulus including the padding bit
-                    u64::MAX % (1 << (block_modulus + 1))
+                    u64::MAX % (block_modulus * 2)
```

Comment:
this can't be equivalent either right ?

Reply:
They are not equivalent, but the form `(1 << (block_modulus + 1))` was incorrect

Reply:
These are a bit hard to detect because the modulus would implicitly be done when generating the lut in `f(i) * delta`

I think this particular case created a problem on GPU but only for 3_3, using hillis steele

Reply:
ok worth pinging the GPU then

Reply:
Guillermo encoutered it today :D